src/client/js/otp/modules/planner/PlannerModule.js

```diff
@@ -122,21 +122,30 @@ otp.modules.planner.PlannerModule =
     
 	getCookie : function()
     {
-		var value = document.cookie.split(';');
```

Comment:
Watch for spaces here - it looks like the standard is "key=value; " and not just ";" so it's possible that not respecting this could break the key= check later on since " visited=" == 1 and not 0

document.cookie.split(';')
Array [ "__utma=203323533.1448402257.1436415…", " __utmz=203323533.1436492951.5.3.ut…", " __atuvc=1%7C30%2C1%7C31%2C1%7C32%2…", " __atssc=google%3B3", " __utmb=203323533.1.10.1440557861", " __utmc=203323533", " __utmt=1", " true", " __atuvs=55dd2b26a3c4dc69000" ]

document.cookie.split('; ')
Array [ "__utma=203323533.1448402257.1436415…", "__utmz=203323533.1436492951.5.3.utm…", "__atuvc=1%7C30%2C1%7C31%2C1%7C32%2C…", "__atssc=google%3B3", "__utmb=203323533.1.10.1440557861", "__utmc=203323533", "__utmt=1", "true", "__atuvs=55dd2b26a3c4dc69000" ]